packages/dotcom-server-handlebars/src/helpers/renderJsxComponent.ts

```diff
@@ -0,0 +1,36 @@
+import path from 'path'
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+
+export default function renderJsxComponent({ hash }) {
+  let modulePath
+
+  if (hash.hasOwnProperty('package')) {
+    modulePath = path.join(process.cwd(), `node_modules/${hash.package}`)
+  }
+
+  // localRoute is relative to root of app consuming dotcom-server-handlebars.
+  if (hash.hasOwnProperty('localRoute')) {
+    modulePath = path.resolve(process.cwd(), `./${hash.localRoute}`)
+  }
+
+  if (!modulePath) {
+    throw new Error('You must specify a "package" or "localRoute" argument to load a module')
+  }
+
+  const importedModule = require(modulePath)
+
+  const Component = hash.hasOwnProperty('namedExport')
+    ? importedModule[hash.namedExport]
+    : importedModule.default
```

Comment:
Why is `default` required? It is not required in x-dash: https://github.com/Financial-Times/x-dash/blob/master/packages/x-handlebars/index.js#L38.

Reply:
I definitely think we should check!

This can be done quite simply:

```js
  const Component = hash.hasOwnProperty('namedExport')
    ? importedModule[hash.namedExport]
    : importedModule && importedModule.__esModule ? importedModule['default'] : importedModule
```

Reply:
Reference: https://github.com/Financial-Times/dotcom-page-kit/blob/a25325d2e534895ac386c508af43a50e9ccf4b56/packages/dotcom-server-react-jsx/src/interopRequire.ts

(This is exactly what TypeScript, Babel, et al all do)

Reply:
Fixed in [this commit](https://github.com/Financial-Times/dotcom-page-kit/pull/746/commits/c04c77832aeea9107ad6451a1443e5c1a9681d20), although Prettier had made it pretty ugly…